src/style.css

```diff
@@ -2,2 +1,70 @@
+.ttq {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.nav_item {
+  display: inline-block;
+  margin-left: 20px;
+}
```

Comment:
You have added a left margin to your '.nav_item' class. This can cause inconsistent spacing because it adds space to the left side of each item, including the first one. It's recommended to be consistent with your vertical margins and add only top or only bottom margin, but not both. In this case, you can add a right margin to each item and remove the left margin from the first child.